### DISCUSSION TITLE
in FORK if one task failed, other task should continue in execution

### DISCUSSION THREAD

**Discussion by bhawani021** [Netflix/conductor#2346] [Q&A]
In current senario, in FORK task if one task failed all other tasks will stop execution and will show as failed.

Which should not be a case.

Please follow ticket for more detail: https://github.com/Netflix/conductor/issues/2337

**Comment by apanicker-nflx** [comment]
Following up on [this comment ](https://github.com/Netflix/conductor/issues/2337#issuecomment-875711846)
```
Fork

SubWf1                |    SubWf2                |   SubWf3
(Task: A, D, G)    |   (Task: B, E, H)      |  (Task: C, F, I)

Join (SubWf1, SubWf2, SubWf3)
```
>What if sub-workflow SubWf3 failed? will SubWf1 and SubWf2 continue to execute?

In this case,  if you mark your sub workflow tasks as optional (set "optional": true) then the subwf1 and subwf2 would continue to execute eve if subwf3 failed. Please note that, subwf3 task in the parent workflow would be marked as COMPLETED_WITH_ERRORS status.
